awx/api/views/__init__.py

```diff
@@ -617,6 +638,23 @@ class ScheduleCredentialsList(LaunchConfigCredentialsBase):
     parent_model = models.Schedule
 
 
+class ScheduleLabelsList(LabelList):
```

Comment:
This view is a sublist-create-attach view. That means that you can create a label by

POST to `/api/v2/schedule/N/labels/`

@mabashian does the UI use this to create labels? Or does it go create them by POST to `/api/v2/labels/`? If the UI does not use it, we should change the views from a create-attach-sublist to just an attach-sublist type view. Otherwise we have this un-tested creation mechanism hanging around, and a multitude of things may be wrong about it.

Same goes for the instance groups sublist.